clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/security/KubernetesV2Credentials.java

```diff
@@ -497,11 +505,14 @@ public void resumeRollout(KubernetesKind kind, String namespace, String name) {
     runAndRecordMetrics("resumeRollout", kind, namespace, () -> jobExecutor.resumeRollout(this, kind, namespace, name));
   }
 
-  public void patch(KubernetesKind kind, String namespace, String name, KubernetesPatchOptions options,
-    KubernetesManifest manifest) {
+  public void patch(KubernetesKind kind, String namespace, String name, KubernetesPatchOptions options, KubernetesManifest manifest) {
     runAndRecordMetrics("patch", kind, namespace, () -> jobExecutor.patch(this, kind, namespace, name, options, manifest));
   }
 
+  public void patch(KubernetesKind kind, String namespace, String name, KubernetesPatchOptions options, List<JsonPatch> patches) {
+    runAndRecordMetrics("patch", kind, namespace, () -> jobExecutor.patch(this, kind, namespace, name, options, patches));
```

Comment:
@maggieneterval -- we will want to add a third way to patch here, and decide which to use depending on what the patch strategy is (json, merge, or strategic).